.pre-commit-config.yaml

```diff
@@ -0,0 +1,67 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+
+      - id: mixed-line-ending
+
+      - id: end-of-file-fixer
+
+      - id: requirements-txt-fixer
+
+      - id: check-yaml
+        args: ["--allow-multiple-documents"]
+
+      - id: check-xml
+
+      - id: check-merge-conflict
+
+      - id: check-case-conflict
+
+      - id: check-docstring-first
+
+      - id: mixed-line-ending
+        args: ["--fix=auto"]
+
+  - repo: https://github.com/iamthefij/docker-pre-commit
```

Comment:
Can we remove the docker related liniting? would prefer to keep the first pass of this simple.